src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -677,6 +690,17 @@ namespace OpenMS
       results_part[hit_idx].setObservedRT(feature.getRT());
       results_part[hit_idx].setSourceFeatureIndex(feature_index);
       results_part[hit_idx].setObservedIntensity(feature.getIntensity());
+      
+      std::vector<double> mti;
+      for (Size i=0; i<3; ++i)
+      {
+        if (feature.metaValueExists("masstrace_intensity_" + String(i)))
```

Comment:
sorry, I dont quite understand how you know that there will be 3 values. Are these isotopes and is this the aggregate intensity across all retention time points? 

also it seems somewhat an abuse to store this in a meta value ...